domain-sample-data/src/main/java/uk/gov/hmcts/cmc/domain/models/sampledata/SampleClaim.java

```diff
@@ -474,7 +515,55 @@ public SampleClaim withSealedClaimDocument(URI sealedClaimDocument) {
         ClaimDocument claimDocument = ClaimDocument.builder()
             .documentManagementUrl(sealedClaimDocument)
             .documentName("001CLAIM-FORM")
-            .documentType(ClaimDocumentType.SEALED_CLAIM)
+            .documentType(SEALED_CLAIM)
+            .createdDatetime(LocalDateTimeFactory.nowInLocalZone())
+            .createdBy("OCMC")
+            .build();
+        this.claimDocumentCollection.addClaimDocument(claimDocument);
+        return this;
+    }
+
+    public SampleClaim withClaimIssueReceiptDocument(URI uri) {
+        ClaimDocument claimDocument = ClaimDocument.builder()
+            .documentManagementUrl(uri)
+            .documentName("claim-form-claimant-copy.pdf")
+            .documentType(CLAIM_ISSUE_RECEIPT)
+            .createdDatetime(LocalDateTimeFactory.nowInLocalZone())
+            .createdBy("OCMC")
```

Comment:
suggestion - "OCMC" to be a constants. Used in many places.

Reply:
done